src/Stripe.net/Services/Payouts/StripePayoutService.cs

```diff
@@ -9,6 +9,10 @@ public class StripePayoutService : StripeBasicService<StripePayout>
     {
```

Comment:
I've had about 5 new ideas for how to handle expandables so there isn't as much cookie cutter. My fear is this is an easy way for the developer to use it, though.